core/src/layers/prometheus.rs

```diff
@@ -88,12 +88,32 @@ use crate::*;
 #[derive(Default, Debug, Clone)]
 pub struct PrometheusLayer {
     registry: Registry,
+    requests_duration_seconds_buckets: Vec<f64>,
+    bytes_total_buckets: Vec<f64>,
 }
 
 impl PrometheusLayer {
     /// create PrometheusLayer by incoming registry.
-    pub fn with_registry(registry: Registry) -> Self {
-        Self { registry }
+    pub fn with_registry(
```

Comment:
Please don't change existing API. We should provide a good default value.

Let's add two new APIs that:

- `pub fn requests_duration_seconds_buckets(mut self, buckets: Vec<f64>) -> Self {}`
- `pub fn bytes_total_buckets(mut self, buckets: Vec<f64>) -> Self {}`

Users will use them like this way:

```rust
PrometheusLayer::with_registry(reg)
  .requests_duration_seconds_buckets(vec![1.0])
  .bytes_total_buckets(vec![1.0])
```

Reply:
In my initial thoughts, I want to remind people there may be some critical arguments you must take care of by changing the function signature.

But I think it is better to keep the API consistent. 

I will add a doc to describe the bucket behavior in the PR